packages/salesforcedx-vscode-apex-replay-debugger/src/messages/i18n.ts

```diff
@@ -40,6 +40,10 @@ export const messages = {
     'Step 5 of 6: Uploading checkpoints',
   checkpoint_creation_status_processing_complete_success:
     'Step 6 of 6: Confirming successful checkpoint creation',
+  checkpoint_upload_in_progress:
+    'A checkpoint update is currently in progress for your org. Wait for the update to complete before you add or remove checkpoints.',
+  checkpoint_upload_error_wrap_up_message:
+    'Your checkpoints have errors. Fix the errors listed in the output, then run SFDX: Update Checkpoints in Org again.',
```

Comment:
You can replace `SFDX: Update Checkpoints in Org` with `%s`. When you call `nls.localize` with this string, you can pass in an additional param that contains the localized string for that command.

Reply:
It'll be done.